src/main/api/2/studio-api-2.yaml

```diff
@@ -3787,6 +3946,76 @@ components:
       type: object
       description: API v1 object
 
+    PublishingPackage:
+      type: object
+      properties:
+        id:
+          type: string
+          description: package ID
+        schedule:
+          type: string
+          format: datetime
+          description: scheduled date and time
+        approver:
+          type: string
+          description: user that approved publishing package
+        state:
+          type: string
+          description: package state
+        environment:
+          type: string
+          description: environment where package will be published
+        comment:
+          type: string
+          description: submission comment
+      required:
+        - id
+        - schedule
+        - approver
+        - state
+        - environment
+
+    PublishingPackageDetails:
+      type: object
+      properties:
+        id:
+          type: string
+          description: package ID
+        schedule:
+          type: string
+          format: datetime
+          description: scheduled date and time
+        approver:
+          type: string
+          description: user that approved publishing package
+        state:
+          type: string
+          description: package state
+        environment:
+          type: string
+          description: environment where package will be published
+        comment:
+          type: string
+          description: submission comment
+        items:
+          type: array
+          items:
+            type: object
+            properties:
+              path:
+                type: string
+                description: content item path
+              contentTypeClass:
```

Comment:
Should we have two things: Type and MIME-type as separate things to avoid confusion?

Reply:
For page/component descriptors it is `page` or `component`, for assets it is mime-type. We are already using the same approach for `getContentItem`. Maybe worth separating since Content Service API v2 is pending.

Reply:
Let's separate them, please.

Reply:
done